README.md

```diff
@@ -7,3 +16,4 @@
 * Bilingual Spanish/English
 * Senior Banker
 * Public Notary
+ master
```

Comment:
When we are presented with a conflict in git we get these labels to tell us which branch the conflicting changes are from. As part of resolving the conflict we'd want to remove these since they are just conflict labels, we wouldn't want them in our code, they might even cause the system to crash.